performant-software/textlab
# Adds spinner, asset pipeline and route changes

**What this PR Does**
1. Adds a wait modal for the editor, allowing feedback 
Closes #155 
Closes #132 

2. Some significant cleanup and back-end changes. See NB below. 

**How to Test**
Go here: https://mel-textlab-pr-186.herokuapp.com/

To close issues #132 and #155, Please log in and click on various projects and confirm the spinner appears while loading. Please also test this with lightweight and empty projects.

NB: Completing the work started with the previous release (and in support of the spinner) I have overhauled the asset pipeline, trimmed some confusing boilerplate code, and adjusted the home routing, which had previously been causing problems with logging out and hitting the site directly in a logged-in versus logged-out site. Since this work basically touches ALL of the CSS and JS for the entire site, I would like to request a full regression test: please try all the features and confirm that the site is working as expected (except for the unaddressed issues already on the board). Thank you!
